docs/python-installation.md

```diff
@@ -110,26 +110,14 @@ The following environment variables need to be configured when instrumenting you
 | `$SCOPE_REPOSITORY`  | Autodetected (\*) | Repository URL to use when sending data to Scope                  |
 | `$SCOPE_SOURCE_ROOT` | Autodetected (\*) | Repository root path                                              |
 
-The following optional parameters can also be configured:
-
-| Environment variable | Default value | Description                                    |
-| -------------------- | ------------- | ---------------------------------------------- |
-| `$SCOPE_SERVICE`     | `default`     | Service name to use when sending data to Scope |
-
 (\*) Autodetection of git information works if either tests run on a [supported CI provider](python-compatibility.md#ci-providers),
 or if the `.git` folder is present locally, and there is an `origin` remote configured pointing to the right repository.
 
-These configuration settings can also be provided via CLI flags. Check out the
-[CLI reference](https://scope-python-agent.readthedocs.io/en/latest/cli.html) documentation for details.
-
-The following environment variables are also available to modify the Scope Agent behavior.
-
-| Environment variable       | Default           | Description                                                                                                                          |
-| -------------------------- | ----------------- | ------------------------------------------------------------------------------------------------------------------------------------ |
-| `$SCOPE_AUTO_INSTRUMENT`   | `true`            | Boolean flag to apply Scope auto instrumentation                                                                                     |
-| `$SCOPE_SET_GLOBAL_TRACER` | `false`           | Boolean flag to register `ScopeTracer` as OpenTracing's global tracer                                                                |
-| `$SCOPE_TESTING_MODE`      | Autodetected (\*) | Boolean flag to indicate to `ScopeAgent` if it's running tests (`true`), or if it's being used for runtime instrumentation (`false`) |
```

Comment:
why is this removed?

Reply:
It should have been  moved to the configuration section